<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Offers</class>
 <widget class="QDialog" name="Offers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1009</width>
        <height>527</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="Offer" name="Off2">
           <property name="font">
            <font>
             <family>Ravie</family>
             <pointsize>36</pointsize>
            </font>
           </property>
           <property name="text">
            <string>MONAKO</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Offer" name="Off3">
           <property name="font">
            <font>
             <family>Ravie</family>
             <pointsize>36</pointsize>
            </font>
           </property>
           <property name="text">
            <string>NOWY YORK</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Offer" name="Off1">
           <property name="font">
            <font>
             <family>Ravie</family>
             <pointsize>36</pointsize>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="mouseTracking">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>KOZIA WOLA</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="Return">
     <property name="styleSheet">
      <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
     </property>
     <property name="text">
      <string>Powrót</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="styleSheet">
      <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
     </property>
     <property name="text">
      <string>Dodaj Ofertę*</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Offer</class>
   <extends>QPushButton</extends>
   <header>OfferObjects.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
